### PR TITLE
Procedure

### PR DESCRIPTION
**Pull request by ZviBaratz**
Made some changes mostly to better accommodate the FE.

### CODE REVIEW

**Comment by ZviBaratz** [comment]
:warning: @Aharonyn

File: research/filters/procedure_filter.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            ("exact", "Exact"),
+        ]
+    )
+    description = filters.LookupChoiceFilter(
+        lookup_choices=[
+            ("contains", "Contains (case-sensitive)"),
+            ("icontains", "Contains (case-insensitive)"),
+            ("exact", "Exact"),
+        ]
+    )
```
Comment (Aharonyn) [inline research/filters/procedure_filter.py]: Why not use the `LOOKUP_CHOICES` from `research.filters.utils`?

File: research/filters/study_filter.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+            ("exact", "Exact"),
+        ]
+    )
+    description = filters.LookupChoiceFilter(
+        lookup_choices=[
+            ("contains", "Contains (case-sensitive)"),
+            ("icontains", "Contains (case-insensitive)"),
+            ("exact", "Exact"),
+        ]
+    )
```
Comment (Aharonyn) [inline research/filters/study_filter.py]: Why not use `LOOKUP_CHOICES` from `research.filters.utils`?

**Review by Aharonyn — COMMENTED** [review]
Looks great!

File: research/filters/procedure_filter.py (hunk 1 above)
Comment (ZviBaratz) [reply research/filters/procedure_filter.py]: No good reason, didn't have the time to refactor. I'll merge to try and set up things on the server and then we can make improvements.

**Pull request merged**